auth_backends/tests/test_backends.py

```diff
@@ -302,4 +302,4 @@ def test_end_session_url(self):
         # NOTE (CCB): We use the strategy's method, rather than override_settings, because the TestStrategy
         # class being used does not rely on Django settings.
         self.strategy.set_settings({'SOCIAL_AUTH_{}_PUBLIC_URL_ROOT'.format(self.name): expected})
-        self.assertEqual(self.backend.end_session_url(), self.url_root + logout_location)
+        self.assertEqual(self.backend.end_session_url(), expected + logout_location)
```

Comment:
@pwnage101 Is this the test change you intended?

Reply:
yep.